blocks/shopify/includes/options/store/class-store.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Store
+ *
+ * @package Cata\Blocks\Shopify\Options
+ * @since 0.8.2
+ */
+
+namespace Cata\Blocks\Shopify\Options;
+
+use Cata\Blocks\Shopify\Options\Add_Store;
+
+/**
+ * Store
+ */
+class Store {
+
+	/**
+	 * Setting Name
+	 * Unique key for a single option/setting, stored in DB.
+	 */
+	const SETTING_NAME = 'cata_blocks_shopify_stores';
+
+	/**
+	 * Register
+	 * 
+	 * @param string $option_group
+	 */
+	public static function register( string $option_group ) : void {
+		register_setting(
+			$option_group,
+			self::SETTING_NAME,
+			array(
+				'type'              => 'array',
+				'sanitize_callback' => array( __CLASS__, 'sanitize_settings' ),
+				'show_in_rest'      => false,
+				'default'           => array(),
+			)
+		);
+
+		add_settings_field(
+			self::SETTING_NAME,
+			'Shopify Stores',
+			array( __CLASS__, 'field_callback' ),
+			$option_group,
+			$option_group . '_section',
+			array(
+				'label_for' => self::SETTING_NAME,
+				'id'        => self::SETTING_NAME,
+				'name'      => self::SETTING_NAME,
+			)
+		);
+	}
+
+	/**
+	 * Sanitize Settings
+	 * 
+	 * @param array $array
+	 * @return array
+	 */
+	public static function sanitize_settings( $array ) : array {
+		if ( empty( $array ) ) {
+			return array();
+		}
+
+		$array = map_deep( $array, 'sanitize_text_field' );
+
+		$array = array_filter( $array, function( $value ) {
+			return ! empty( $value['subdomain'] ) && ! empty( $value['access_token'] );
+		} );
+
+		return $array;
+	}
+
+	/**
+	 * Field Callback
+	 * 
+	 * @param array $args Everything you need for an input.
+	 */
+	public static function field_callback( array $args ) : void {
+		$key = 0;
+		$setting = get_option( self::SETTING_NAME, array() );
+		$num_options = count( $setting );
+
+		?>
+			<div id="<?php echo esc_attr( self::SETTING_NAME ) . '_wrapper'; ?>" style="margin-bottom: 2em;">
+		<?php
+			do {
+				$option = array_key_exists( $key, $setting ) ? $setting[$key] : array();
+				$subdomain = is_array( $option ) && array_key_exists( 'subdomain', $option ) ? $option['subdomain'] : '';
+				$access_token = is_array( $option ) && array_key_exists( 'access_token', $option ) ? $option['access_token'] : '';
+				echo $key > 0 ? '<br>' : '';
```

Comment:
Could this conditional line-break be removed if the `fieldset` elements here and in the JS for add-store all had `style="margin-bottom: 2em;"` instead of the wrapper div?

Reply:
Yes using margin instead works here!